src/main/java/BMO/Ui.java

```diff
@@ -0,0 +1,112 @@
+package bmo;
+
+import java.util.ArrayList;
+
+import bmo.task.Task;
+
+/**
+ * Represents the user interface of BMO.
+ */
+public class Ui {
```

Comment:
Since this class isn't saving instance of any class. Maybe can change it into static class?
what do you think?

Reply:
I'm assuming that when you say static class, you are referring to a nested class. I think that abstracting out the Ui into its own class makes the code more readable since it consolidates all printing methods in one class, rather than cluttering another class with these methods.